app/locale/en.coffee

```diff
@@ -1183,7 +1185,7 @@ module.exports = nativeDescription: "English", englishDescription: "English", tr
     course_suffix: ""
 
   teachers_quote:
-    subtitle: "Get your students started in less than an hour. You'll be able to <strong>create a class, add students, and monitor their progress</strong> as they learn computer science."
+    subtitle: "Learn more about CodeCombat with an interactive walk through of the product!" # {change}
```

Comment:
Highlighting this line as in the Asana task the copy was: "Learn more about CodeCombat with an interactive walkthrough of the product, pricing, and implementation."

Reply:
Thank you